tune_sklearn/_trainable.py

```diff
@@ -104,7 +105,10 @@ def _early_stopping_partial_fit(self, i, estimator, X_train, y_train):
         """Handles early stopping on estimators that support `partial_fit`.
 
         """
-        estimator.partial_fit(X_train, y_train, np.unique(self.y))
+        try:
+            estimator.partial_fit(X_train, y_train, classes=np.unique(self.y))
+        except AttributeError:
```

Comment:
Where does this `AttributeError` come from?

Reply:
Some estimators don't support classes in partial fit.

Reply:
hmm @Yard1 how about using `inspect.getfullargspec` to detect if classes is in the method signature rather than eagerly executing?

Reply:
I don't think there is any benefit to what you propose. There aren't going to be any side effects from the execution as the AttributeError will be raised before the function body. Using `inspect` just seems overkill here, especially considering there are no other attributes that could raise this exception. Plus this way it's kept simple and without the need for importing anything.

Reply:
Well the benefit is that if the user's estimator.partial_fit is implemented incorrectly, any underlying attribute error will not be accidentally caught. 

inspect is stdlib so there's no harm in importing it.

Reply:
AFAIK inspect is the standard way of doing this - https://stackoverflow.com/questions/23228664/how-to-check-which-arguments-a-function-method-takes

Reply:
Very well then, will do 👍